sessiontxn/isolation/readcommitted_test.go

```diff
@@ -504,7 +504,7 @@ func TestConflictErrorsInRC(t *testing.T) {
 	se.SetValue(sessiontxn.AssertLockErr, nil)
 	tk.MustExec("delete from t where id = 1")
 	_, ok = se.Value(sessiontxn.AssertLockErr).(map[string]int)
-	require.False(t, ok)
+	require.Equal(t, ok, true)
```

Comment:
`require.True()` is simpler

Reply:
Yes, but `require.Equal` will give a more clear error message.